src/main/java/me/botsko/prism/database/sql/SqlSelectQueryBuilder.java

```diff
@@ -618,7 +559,8 @@ public QueryResult executeSelect(TimeTaken eventTimer) {
                                 newData = Bukkit.createBlockData(item.getType());
                             } catch (IllegalArgumentException e) {
                                 // This exception occurs, for example, with "ItemStack{DIAMOND_LEGGINGS x 1}"
-                                Prism.debug("IllegalArgumentException for record #" + rowId + " calling createBlockData for " + item.toString());
+                                Prism.debug("IllegalArgumentException for record #" + rowId + " calling createBlockData for "
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 125).](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)